Source/KerbalCombatSystems/DecouplerDesignator.cs

```diff
@@ -1,18 +1,47 @@
-﻿namespace KerbalCombatSystems
+﻿using UnityEngine;
+
+namespace KerbalCombatSystems
 {
     public class ModuleDecouplerDesignate : PartModule
     {
-        const string DecouplerDesignationGroupName = "Decoupler Designation";
+        //denote seperator type
+        [KSPField(isPersistant = true)]
+            public string seperatorType = "seperatorType";
+
 
+        [KSPField(isPersistant = true)]
+            public bool seperated = false;
+
+        const string DecouplerDesignationGroupName = "Seperator Designation";
```

Comment:
```suggestion
        const string DecouplerDesignationGroupName = "KCS Designation";
```

What do you think of this rename? I'm not sure myself. I think it would be good to let the user know where the extra module is coming from though, otherwise I don't think it's immediately obvious what it's for.

![image](https://user-images.githubusercontent.com/30965946/204271593-02031bc0-2c70-45cc-9ec3-d57f66962631.png)

Reply:
Saying separator twice is a bit redundant in fairness